UnityProject/Assets/Scripts/Systems/Inventory/ItemStorage.cs

```diff
@@ -158,13 +160,25 @@ public bool ServerTryTransferFrom(ItemSlot inSlot)
 
 	public bool ServerTryRemove(GameObject InGameObject, bool Destroy = false)
 	{
-		var Item = InGameObject.GetComponent<ItemAttributesV2>();
+		ItemAttributesV2 Item = InGameObject.GetComponent<ItemAttributesV2>();
```

Comment:
```suggestion
		var item = InGameObject.GetComponent<ItemAttributesV2>();
````
remember to change all uses of the variable as well

Reply:
I'm not following, what's wrong with this?